node/forcedroid.js

```diff
@@ -462,7 +462,17 @@ function fixSdkGradleFiles(config) {
 
 function fixSdkGradleFileHelper(appFolderName, lib) {
     console.log('Tweaking build.gradle for library ' + lib);
-    shelljs.sed('-i', "compile project(':libs:", "compile project(':forcedroid:libs:", path.join(appFolderName, "forcedroid", "libs", lib, "build.gradle"));    
+    replaceInFile(path.join(appFolderName, "forcedroid", "libs", lib, "build.gradle"), "compile project(':libs:", "compile project(':forcedroid:libs:"); 
+}
+
+function replaceInFile(fileName, textInFile, replacementText) {
+    var contents = fs.readFileSync(fileName, 'utf8');
+    var lines = contents.split(/\r*\n/);
+    var result = lines.map(function (line) {
+      return line.replace(textInFile, replacementText);
+    }).join('\n');
```

Comment:
That sounds like a good idea. I'll close this PR. Will send a new PR for the shared repo and one more here with updated submodule pointer to shared repo and code changes in forcedroid.js I'll move this function to node/util.js